app.js

```diff
@@ -0,0 +1,16 @@
+const hamburgerOpenConst = document.querySelector('.hamburgerOpen');
+const hamburgerCloseConst = document.querySelector('.hamburgerClose');
+const menuPageConst = document.querySelector('.menuPage');
+// const pageLinkConst = document.querySelectorAll('.pageLink');
+
+function open() {
+  menuPageConst.style.display = 'flex';
+  menuPageConst.style.top = '0';
+}
+
+function close() {
```

Comment:
- [ ] You have done great so far, but there are some requirements you still have to implement before moving to the next milestone :+1: :+1: 

- [ ] As stated in the project requirement, you need to try to replicate the Figma design as much as you can, Currently, your Hamburger icon position doesn't match the one in the Figma design, Looking at the Figma you can see that there is a space at the right of the icon but that's not the case In your project. I'll suggest you update this, 
- [ ] Also there seems to be a white space at the right of your mobile design, I'll suggest you look into this too. :+1: :smile: 

- - Your current design:
![Selection_077](https://github.com/bhushan354/Portfolio_Project/assets/107632780/29553314-511d-4b16-a4a8-bad185affbcb)

- - Figma design:
![Screenshot from 2023-06-26 15-05-02](https://github.com/bhushan354/Portfolio_Project/assets/107632780/9a1332c2-76f5-437a-9a24-3cbe639536ba)

Reply:
Hello @V-Blaze ,
I have been trying to solve this issue for a long time. can you please tell me with more context how do I solve this issue.

thank you.